support-frontend/assets/pages/paper-subscription-landing/components/tabs.jsx

```diff
@@ -38,27 +40,38 @@ type DispatchPropTypes = {|
 type PropTypes = {|
   ...StatePropTypes,
   ...DispatchPropTypes,
+  useDigitalVoucher: Option<boolean>,
 |};
 
+const getTabTitle = (useDigitalVoucher, k) => {
+  if (k === 'HomeDelivery' || useDigitalVoucher) {
```

Comment:
what is `k`?

Reply:
I see that it's "key" now but it must mean tabName or something?
I assume the exception is Voucher, so I'd prefer to see that as the exception, as it looks like Home delivery or use digital voucher is the exception
```
if (k === 'Voucher' && useDigitalVoucher) {
 return 'Voucher Booklet';
} 
```

Reply:
For the moment, the voucher booklet actually is the default. But this will change, so I'm torn! We will probably refactor this code in a bit once we're sure we don't need to switch vouchers back on. I guess that will be a good opportunity to get rid of some of this complexity.

You're so right about k. That has to go!